content/en/docs/setup/install/multicluster/simplified/index.md

```diff
@@ -0,0 +1,168 @@
+with a third cluster that's on a different network.
+
+Using the approach shown in this guide results in an instance of the Istio control plane being deployed in every cluster
+within the mesh. Although this is a common configuration, other more complex topologies are possible, but have to be done
+using a more manual process, not described herein.
+
+## Before you begin
+
+The procedures we describe here are primarily intended to be used with relatively pristine clusters,
+where Istio hasn't already been deployed. We hope to expand support in the future to existing cluaters.
+
+For the sake of explanation, this guide assumes you have created three Kubernetes clusters:
+
+- A cluster named `cluster-east-1` on the network named `network-east`.
+- A cluster named `cluster-east-2` on the network named `network-east`.
+- A cluster named `cluster-west-1` on the network named `network-west`.
+
+These clusters shouldn't have Istio on them yet. The first two clusters are on the same network and have
+direct connectivity, while the three cluster is on a different network.
+Take a look at the [platform setup instructions](/docs/setup/platform-setup)
+for any special instructions for your particular environment.
+
+## Initial preparations
+
+You need to do a few one-time steps in order to be able to setup a multicluster mesh:
+
+1. Ensure that all of your clusters are included in your [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#define-clusters-users-and-contexts)
+and create contexts for each cluster. Once you're done, your configuration file should include something similar to:
+
+    {{< text syntax="yaml" downloadas="kubeconfig.yaml" >}}
+    kind: Config
+    apiVersion: v1
+    clusters:
+    - cluster:
+      name: cluster-east-1
+    - cluster:
+      name: cluster-east-2
+    - cluster:
+      name: cluster-west-1
+    contexts:
+    - context:
+        cluster: cluster-east-1
+      name: context-east-1
+    - context:
+        cluster: cluster-east-2
+      name: context-east-2
+    - context:
+        cluster: cluster-west-1
+      name: context-west-1
+    {{< /text >}}    
+
+1. Decide on what the name of your multicluster mesh will be. Something short but memorable is your best choice here:
+
+    {{< text bash >}}
+    $ export MESH_ID=mymeshname
+    {{< /text >}}
+
+1. Decide on the organization name to use in the root and intermediate certificates created to let the clusters communicate with one
+another. This should generally be derived from your organization's DNS name:
+
+    {{< text bash >}}
+    $ export ORG_NAME=mymeshname.mycompanyname.com
+    {{< /text >}}
+
+1. Create a working directory where to store a number of files produced during the cluster
+onboarding process:
+
+    {{< text bash >}}
+    $ export WORKDIR=mydir
+    $ mkdir -p ${WORKDIR}
+    $ cd ${WORKDIR}
+    {{< /text >}}
+
+1. Download the setup script to your working directory:
```

Comment:
Seems like there should be some kind of overview of what the script does. Maybe in the sample README.md and then link to it from here?

Reply:
Done.